outdoorsmanship.lic

```diff
@@ -24,30 +26,42 @@ class Outdoorsmanship
     @training_spells = @settings.crafting_training_spells
     @targetxp = 4
     @targetxp = args.mindstates.to_i if args.mindstates
+    @outdoors_room = @settings.outdoors_room
+    if args.room
+      room = args.room.split('=')[1]
+      @outdoors_room = room
+    end
+
+    @train_method = args.forage || @settings.outdoors_method
 
     @forage_item = if args.collect_item
                      args.collect_item
-                   else
+                   elsif @settings.forage_item
                      @settings.forage_item
+                   elsif DRSkill.getrank('Outdoorsmanship') < 20
+                     'rock'
                    end
+
     if args.perception
       @skill_name = 'Perception'
     else
       @skill_name = 'Outdoorsmanship'
     end
+
     start_exp = DRSkill.getxp(@skill_name)
-    @end_exp = [start_exp + @targetxp, 29].min
+    @end_exp = [start_exp + @targetxp, 34].min
 
+    DRCT.walk_to(@outdoors_room) if @outdoors_room
```

Comment:
We should really just put this `if` check in `walk_to` at some point... I feel like we do this guard in a lot of places

Reply:
It might work as-is? If I'm understanding this line correctly, It walks to `outdoors_room` if one is set in the yaml. If we left the `if` off of this, it would call `walk_to(nil)`.  And `DRCT.walk_to` returns false if the target room is nil. So it should do nothing and just continue the script in your current room, I think, giving the same result.

Reply:
Accurate.  Leaving it blank works fine.  Tested!